rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -103,9 +103,9 @@ check_and_fix_cfengine_processes() {
     echo " Done"
   fi
 
-  # Get the value of rudder-agent run interval from file /var/rudder/cfengine-community/inputs/run_interval
-  if [ -f "${CFE_DIR}/inputs/run_interval" ]; then
-    RUN_INTERVAL=`cat "${CFE_DIR}/inputs/run_interval"`
+  # Get the value of rudder-agent run interval from file /var/rudder/cfengine-community/inputs/run-interval
+  if [ -f "${CFE_DIR}/inputs/run-interval" ]; then
+    RUN_INTERVAL=`cat "${CFE_DIR}/inputs/run-interval"`
```

Comment:
I will do this in another PR to make it easier to read/merge.
See http://www.rudder-project.org/redmine/issues/8814